test/units/testsuite-64.sh

```diff
@@ -237,17 +254,42 @@ EOF
     # On unpatched udev versions the delete-recreate cycle may trigger a race
     # leading to dead symlinks in /dev/disk/
     for ((i = 1; i <= iterations; i++)); do
-        udevadm lock --device="$blockdev" sfdisk -q --delete "$blockdev"
-        udevadm lock --device="$blockdev" sfdisk -q -X gpt "$blockdev" <"$partscript"
+        for disk in {0..9}; do
+            if ((disk % 2 == i % 2)); then
+                udevadm lock --device="${devices[$disk]}" sfdisk -q --delete "${devices[$disk]}" &
+            else
+                udevadm lock --device="${devices[$disk]}" sfdisk -q -X gpt "${devices[$disk]}" <"$partscript" &
+            fi
+        done
+
+        # Wait for the above sfdisk commands.
+        for disk in {0..9}; do
+            udevadm lock --device="${devices[$disk]}" true
```

Comment:
I'm just curious - why is the `udevadm lock` call needed here when we called `sfdisk` above with `udevadm lock` as well?

Reply:
The comment here lacks important words: Wait for the above sfdisk commands _to be finished_.